src/Annotations/AppKernel.php

```diff
@@ -48,7 +48,10 @@
     public function __construct(string $environment = 'dev')
     {
         $configFile = __DIR__.'/../../config.php';
-        $config = array_merge(file_exists($configFile) ? require $configFile : [], require __DIR__."/../../config/{$environment}.php");
+        $config = array_merge(
+            require __DIR__."/../../config/{$environment}.php",
+            file_exists($configFile) ? require $configFile : []
```

Comment:
Is the order change meant?

Reply:
Yes, because otherwise it's not possibly to override an environment's committed file with the provisioned one.

Reply:
What's the use case? (Can understand `dev` for trying stuff out, but sounds dangerous for any other environment.)

Reply:
(`container.php` `aws` setting would be ignored)

Reply:
Problem to me looks to be that `dev` has hard-coded configuration.

Reply:
It seems to contain the defaults for the Vagrant VM, which are also the defaults for running the project locally with the help of `goaws`. In fact, `end2end`|`continuumtest`|`prod` are all empty as everything is provided by the provisioned `config.php`.

Reply:
It doesn't make much of a difference now because the `config/$env.php` files are almost empty. I'll revert it and later I'll remove everything remaining in them until this only loads `config.php`